.circleci/config.yml

```diff
@@ -146,8 +150,8 @@ install_dep_180: &install_dep_180
         pip install --progress-bar off  git+https://github.com/min-xu-ai/torch_pg.git@c723ab4#egg=torch-pg
         python -c 'import torch; print("Torch version:", torch.__version__)'
         python -c 'import torch; assert torch.__version__.split(".")[:2] == ["1", "8"], "wrong torch version"'
-        pip list|grep torch
         python -m torch.utils.collect_env
+        wget -O /home/circleci/venv/check_version.py https://raw.githubusercontent.com/min-xu-ai/check_verion/main/check_version.py
```

Comment:
typo, no ? 'check_verion'

Reply:
actually, it is not. It is how I made my repo. (by accident)